main.go

```diff
@@ -118,6 +118,7 @@ func mainWithError() error {
 	daemonCommand.PersistentFlags().String(f.Guest.Cluster.Calico.Subnet, "", "Network address for the CIDR block used by Calico.")
 	daemonCommand.PersistentFlags().String(f.Guest.Cluster.Kubernetes.API.ClusterIPRange, "", "CIDR Range for Pods in cluster.")
 	daemonCommand.PersistentFlags().String(f.Guest.Cluster.Vault.Certificate.TTL, "", "Vault certificate TTL.")
+	daemonCommand.PersistentFlags().String(f.Service.Cluster.Address, "http://localhost:8089", "http://<host>:<port> of the cluster service.")
```

Comment:
IMO this should be `f.Service.ClusterService.Address`. My understanding is `.Service` key is this "service" configuration. Just to make sure it doesn't collide with things like `.Server`. `Cluster.Address` makes me think it's kubernetes address.

Reply:
Ack. I'll change this. I copied this directly from `api`, but I guess that's not exactly up-to-date there :slightly_smiling_face: